### ISSUE TITLE
App is ****** unremovable

### ISSUE THREAD

**Issue by xtance** [snowie2000/mactype#620]
Please make a "remove from this PC" feature
It's non-existent by now. Thanks! (doing a full install before didn't help)

**Comment by sammilucia** [comment]
Hi @xtance. That's just unnecessary language.

Please install the latest beta6 from GitHub releases https://github.com/snowie2000/mactype/releases ... It will clean up any old versions and add Uninstall via Apps and Features et al.

If you have any problems, please let me know more detail and I'll look into it.

**Comment by xtance** [comment]
The thing is, I installed MacType at separate disk (at first)
Yes I installed latest beta version and then removed it, but these files I downloaded yesterday simply don't go away (even though I installed beta version in the same folder as older one).
Final solution was to choose manual mode so that MacType won't hook into programs and therefore its files can be deleted.

I think this site https://www.mactype.net/ should point to beta version, which is removable, or at least note that it's fucking unremovable. 

Thanks.
